testing/e2e/provisioning/README.md

```diff
@@ -0,0 +1,89 @@
+
+The test is executed on a dev cluster. It is executed after every merge to the `kyma` repository that changes the `compass` chart.
+
+## Prerequisites
+
+To run this test, you must have the following Secrets inside your cluster:
+- Gardener Secret per provider
+- Service Manager Secret
+
+You must also have Kyma Environment Broker [configured](https://github.com/kyma-project/kyma-environment-broker/tree/main#configuration) to use these Secrets in order to successfully create Kyma runtime.
+
+## Details
+
+### End-to-end provisioning test
+The provisioning end-to-end test contains a broker client implementation which mocks Registry. It is an external dependency that calls the broker in the regular scenario. The test is divided into two phases:
+
+1. Provisioning
+
+    During the provisioning phase, the test executes the following steps:
+
+    a. Sends a call to KEB to provision a Runtime. KEB creates an operation and sends a request to the Runtime Provisioner. The test waits until the operation is successful. It takes about 30 minutes on GCP and a few hours on Azure. You can configure the timeout using the environment variable.
+
+    b. Creates a ConfigMap with **instanceId** specified.
+
+    c. Fetches the DashboardURL from KEB. To do so, the Runtime must be successfully provisioned and registered in the Director.
+
+    d. Updates the ConfigMap with **dashboardUrl** field.
+
+    e. Creates a Secret with a kubeconfig of the provisioned Runtime.
+
+    f. Ensures that the DashboardURL redirects to the UUA login page. It means that the Kyma Runtime is accessible.
+
+2. Cleanup
+
+    The cleanup logic is executed at the end of the end-to-end test or when the provisioning phase fails. During this phase, the test executes the following steps:
+
+    a. Gets **instanceId** from the ConfigMap.
+
+    b. Removes the test's Secret and ConfigMap.
+
+    c. Fetches the Runtime kubeconfig from the Runtime Provisioner and uses it to clean resources which block the cluster from deprovisioning.
+
+    d. Sends a request to deprovision the Runtime to KEB. The request is passed to the Runtime Provisioner which deprovisions the Runtime.
+
+    e. Waits until the deprovisioning is successful. It takes about 20 minutes to complete. You can configure the timeout using the environment variable.
+
+Between the end-to-end test phases, you can execute your own test directly on the provisioned Runtime. To do so, use a kubeconfig stored in a Secret created in the provisioning phase.
+
+### End-to-end suspension test
+
+The end-to-end suspension test uses the **Trial** Service Plan ID to provision Kyma Runtime. Then, the test suspends and unsuspends the Kyma Runtime and ensures that it is still accessible after the process. The suspension test works similarly to the provisioning test, but it has two additional steps in the `Provisioning` phase:
```

Comment:
Trial, like every plan, has its plan ID in KEB, which is passed during provisioning so that KEB can determine which plan should be provisioned. In this case, the subaccount doesn't matter.